src/components/Layout.vue

```diff
@@ -25,21 +19,50 @@
             hide-footer
             centered
             no-close-on-backdrop
-          >{{ $t('components.layout.sign_wallet', { network: foreignNetworkName }) }}</b-modal>
+            >{{
+              $t("components.layout.sign_wallet", {
+                network: foreignNetworkName,
+              })
+            }}</b-modal
+          >
           <b-modal
             id="already-mapped"
             title="$t('components.layout.account_mapped')"
             hide-footer
             centered
             no-close-on-backdrop
-          >{{ $t('components.layout.already_mapped') }}</b-modal>
+            >{{ $t("components.layout.already_mapped") }}</b-modal
+          >
           <transition name="page" mode="out-in">
             <router-view></router-view>
             <!-- <p class="custom-notification">Scheduled maintance for upgrading to DPOSv3, please check back in a few hours.</p> -->
           </transition>
         </div>
       </div>
     </div>
+    <b-modal
+      no-close-on-backdrop
+      hide-header
+      hide-footer
+      centered
+      v-model="badWalletNetworkConnected"
+    >
+      <span>
+        {{
+          $t("components.layout.your_wallet_connected_to", {
+            network: ethereumNets[foreignNetworkId],
+          })
+        }}
+        {{
+          $t("components.layout.please_change_to", {
+            network: s.ethereum.networkName,
```

Comment:
Change this line to `network: ethereumNets[s.ethereum.networkId],` so the capitalization matches.